backend/hitas/services/job_performance.py

```diff
@@ -76,3 +78,13 @@ def find_job_performance_per_user(source: JobPerformanceSource, start_date: date
         .values("full_name", "job_performance_count")
         .order_by("-job_performance_count")
     )
+
+
+def find_apartment_sales(start_date: datetime.date, end_date: datetime.date):
```

Comment:
Are we finding ApartmentSales in this function? 🤔

Reply:
Yes, creation of those are in LogEntries, no where else